quickstarts/oma-aqm/config.yml

```diff
@@ -1,5 +1,5 @@
 id: ccdf1cf7-29f0-42ca-ae0d-239dfb073fd0
-slug: alert-quality-management
+slug: oma-aqm
```

Comment:
Let's keep this slug the same, otherwise the old link to this quickstart will break. If we really need to change this, we'll need to get something on the backlog to setup redirects.